solidity/contracts/bridge/Bridge.sol

```diff
@@ -1541,6 +1563,72 @@ contract Bridge is
         self.updateTreasury(treasury);
     }
 
+    /// @notice Allows authorized controllers to increase Bank balances via the
+    ///         Bridge.
+    /// @dev "Controllers" here are addresses that have been authorized via
+    ///      `setAuthorizedBalanceIncreaser` and stored in the
+    ///      `authorizedBalanceIncreasers` mapping.
+    /// @param recipient Address receiving the balance increase.
+    /// @param amount Amount by which the balance is increased.
+    function controllerIncreaseBalance(address recipient, uint256 amount)
```

Comment:
I noticed that `controllerIncreaseBalance` only check authorization but don't enforce any amount limits, rate limits, or caps, which means authorized controllers have unrestricted minting power. While the documentation indicates this is intentional and delegates amount validation to controller implementations, I'd like to confirm: is it a requirement that controller contracts implement their own minting limits? If a controller is compromised, the blast radius would be system-wide without Bridge-level circuit breakers. 

I'd consider adding explicit documentation warnings in `IBridgeMintingAuthorization.sol` about this responsibility, or Bridge-level safeguards like per-controller daily mint caps and enhanced monitoring events (Watchdog?).